UnicornRender/video/source/Renderer.cpp

```diff
@@ -23,7 +23,7 @@ Renderer::Renderer(system::Manager& manager, system::Window* window, Camera cons
 {
     if(m_pWindow == nullptr)
     {
-        LOG_ERROR("Window pointer in nullptr!");
+        spdlog::get(utility::Logger::LoggerName())->error("Window pointer in nullptr!");
```

Comment:
`std::shared_ptr<spdlog::logger>` should be a class variable. There is no need to call `get()` every time.